h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -794,46 +765,45 @@ private void fitLSM(Solver s){
     }
 
     private void fitIRLSM(Solver s) {
-      GLMWeightsFun glmw = new GLMWeightsFun(_parms);
-      double [] betaCnd = _state.beta();
+      double[] betaCnd = _state.beta();
       LineSearchSolver ls = null;
       boolean firstIter = true;
       int iterCnt = 0;
       try {
         while (true) {
           iterCnt++;
           long t1 = System.currentTimeMillis();
-          ComputationState.GramXY gram = _state.computeGram(betaCnd,s);
+          ComputationState.GramXY gram = _state.computeGram(betaCnd, s);
           long t2 = System.currentTimeMillis();
           if (!_state._lsNeeded && (Double.isNaN(gram.likelihood) || _state.objective(gram.beta, gram.likelihood) > _state.objective() + _parms._objective_epsilon)) {
             _state._lsNeeded = true;
           } else {
             if (!firstIter && !_state._lsNeeded && !progress(gram.beta, gram.likelihood)) {
-              System.out.println("DONE after " + (iterCnt-1) + " iterations (1)");
+              System.out.println("DONE after " + (iterCnt - 1) + " iterations (1)");
               return;
             }
             betaCnd = s == Solver.COORDINATE_DESCENT?COD_solve(gram,_state._alpha,_state.lambda()):ADMM_solve(gram.gram,gram.xy);
           }
           firstIter = false;
           long t3 = System.currentTimeMillis();
-          if(_state._lsNeeded) {
-            if(ls == null)
+          if (_state._lsNeeded) {
+            if (ls == null)
               ls = (_state.l1pen() == 0 && !_state.activeBC().hasBounds())
-                 ? new MoreThuente(_state.gslvr(),_state.beta(), _state.ginfo())
-                 : new SimpleBacktrackingLS(_state.gslvr(),_state.beta().clone(), _state.l1pen(), _state.ginfo());
+                      ? new MoreThuente(_state.gslvr(), _state.beta(), _state.ginfo())
```

Comment:
It is very hard to read the actual changes because of the formatting.

Reply:
I changed it back to what it was before.